tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -0,0 +1,146 @@
+import uuid
+from datetime import datetime, timedelta
+from typing import Any, Dict
+
+import pytest
+from django.utils import timezone
+
+from sentry.monitors.consumers.check_in import _process_message
+from sentry.monitors.models import (
+    CheckInStatus,
+    Monitor,
+    MonitorCheckIn,
+    MonitorEnvironment,
+    MonitorStatus,
+    MonitorType,
+    ScheduleType,
+)
+from sentry.testutils import TestCase
+from sentry.utils import json
+
+
+class MonitorConsumerTest(TestCase):
+    def get_message(self, monitor_slug: str, **overrides: Any) -> Dict[str, Any]:
+        now = datetime.now()
+        self.guid = uuid.uuid4().hex
+        payload = {
+            "monitor_slug": monitor_slug,
+            "status": "ok",
+            "duration": None,
+            "check_in_id": self.guid,
+            "environment": "production",
+        }
+        payload.update(overrides)
+
+        wrapper = {
+            "start_time": now.timestamp(),
+            "project_id": self.project.id,
+            "payload": json.dumps(payload),
+        }
+
+        return wrapper
+
+    def _create_monitor(self, **kwargs):
+        return Monitor.objects.create(
+            organization_id=self.organization.id,
+            project_id=self.project.id,
+            next_checkin=timezone.now() + timedelta(minutes=1),
+            type=MonitorType.CRON_JOB,
+            config={"schedule": "* * * * *", "schedule_type": ScheduleType.CRONTAB},
+            **kwargs,
+        )
```

Comment:
I think @evanpurkhiser added a helper somewhere else to do this

Reply:
yeah...was in an API testing class I didn't want to import

Reply:
yeah it is in the base API test case

Reply:
Could be worth moving it to our standard place so that it's accessible from all tests (doesn't have to be this pr) https://github.com/getsentry/sentry/blob/f5ab3c97bf6442a8a9cf0b6f835d2a68eaf17736/src/sentry/testutils/fixtures.py#L144-L146

Reply:
👍  I am down for this

Reply:
I'll move it